src/main/java/org/sonarlint/intellij/actions/SonarConfigureProject.java

```diff
@@ -19,17 +19,12 @@
  */
 package org.sonarlint.intellij.actions;
 
+import com.intellij.openapi.actionSystem.AnAction;
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.options.ShowSettingsUtil;
-import com.intellij.openapi.project.Project;
-import org.sonarlint.intellij.analysis.AnalysisStatus;
 import org.sonarlint.intellij.config.project.SonarLintProjectConfigurable;
 
-public class SonarConfigureProject extends AbstractSonarAction {
-  @Override
-  protected boolean isEnabled(AnActionEvent e, Project project, AnalysisStatus status) {
```

Comment:
I don't understand why this part has been removed. Could you elaborate a bit ? Also it's important that all actions keep implementing `AbstractSonarAction` as there is some logic there, for example implementing `UpdateInBackground`.

Reply:
And...I agree again, no reason to remove this implementation with the current state, this is some leftovers of previous tests I did with basic classes :face_in_clouds: